src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
+    private double radius;
+
+    public Circle(String color, double radius) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. For better code readability, assign the radius to a local variable before passing it to the constructor.